src/guiguts/application.py

```diff
@@ -818,6 +822,11 @@ def rowcol_str() -> str:
             "Shift-ButtonRelease-1",
             lambda: preferences.toggle(PrefKey.LINE_NUMBERS),
         )
+        the_statusbar.add_binding(
+            "rowcol",
+            "Shift-ButtonRelease-2",
```

Comment:
I did this, but it's acting weird:
- shift-click works (toggle line numbers)
- shift-ctrl-click works (toggle col numbers)
- shift-(two-finger-tap) - normally equivalent to right-click - sometimes reads as cols, sometimes nums...

I don't have an external mouse handy to try an actual right-click button. The 2-finger tap must have some extra magic we don't know yet. @srjfoo what result do you get?

Reply:
It seems to work properly if I'm not going too fast. Maybe in my earlier test I went too fast and it was interpreted as a double- or triple-click rather than the modified single right-click I wanted. Slowing down it does work.

And the event fired also seems correct.

```
<ButtonPress event state=Shift num=2 x=101 y=8>
<ButtonRelease event state=Shift|Button2 num=2 x=101 y=8>
```

So unless @srjfoo finds the behavior is wrong, I think it's fine.